hadoop-hdds/docs/content/security/SecuringTDE.md

```diff
@@ -71,20 +71,24 @@ will be completely transparent to the clients and end users.
 
 There are two ways to create an encrypted bucket that can be accessed via S3 Gateway.
 
-####1. Create a bucket using shell under "/s3v" volume
+#### Option 1. Create a bucket using shell under "/s3v" volume
 
   ```bash
-  ozone sh bucket create -k encKey /s3v/encryptedBucket
+  ozone sh bucket create -k encKey --layout=FILE_SYSTEM_OPTIMIZED /s3v/encryptedbucket
   ```
-####2. Create a link to an encrypted bucket under "/s3v" volume
+
+#### Option 2. Create a link to an encrypted bucket under "/s3v" volume
 
   ```bash
-  ozone sh bucket create -k encKey /vol/encryptedBucket
-  ozone sh bucket link  /vol/encryptedBucket /s3v/linkencryptedbucket
+  ozone sh bucket create -k encKey --layout=FILE_SYSTEM_OPTIMIZED /vol/encryptedbucket
```

Comment:
Should we set FSO for the bucket which is to accessed via S3?

Reply:
Good point. The default is `OBJECT_STORE` is unspecified, and completely rejects HCFS access.

```bash
$ ozone fs -ls ofs://ozone1/tenant1/encrypted-bucket2/
-ls: Bucket: encrypted-bucket2 has layout: OBJECT_STORE, which does not support file system semantics. Bucket Layout must be FILE_SYSTEM_OPTIMIZED or LEGACY.
```

Note although `LEGACY` is listed above as an option, we no longer allow creating bucket with those types in Ozone Shell anymore.

Reply:
I'm not totally sure of the compatiblity or performance impact if we specify FSO type for S3-only buckets. (FSO doesn't fully support object store IIRC but I'm not exactly sure why, or why that decision is made)

Reply:
> I'm not totally sure of the compatiblity or performance impact if we specify FSO type for S3-only buckets. (FSO doesn't fully support object store IIRC but I'm not exactly sure why, or why that decision is made)

Same here, that's why I asked.

Reply:
Using `OBJECT_STORE` as the default for the S3G example now. And add a long note 2 paragraph with searchable warning message.